backend/src/apis/KlicktippController.ts

```diff
@@ -4,80 +4,135 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
 import CONFIG from '@/config'
+import LogError from '@/server/LogError'
 
 // eslint-disable-next-line import/no-relative-parent-imports
 import KlicktippConnector from 'klicktipp-api'
 
 const klicktippConnector = new KlicktippConnector()
 
-export const klicktippSignIn = async (
+const callKlickTippAPI = (callback: (arg0: any) => any, args: any) => {
+  if (!CONFIG.KLICKTIPP) {
+    return true
+  }
+  return callback(args)
+}
```

Comment:
lets not do this, this makes thinks so much more complicated for a boolean check?!